src/composables/useHotKey/index.js

```diff
@@ -26,7 +27,7 @@ function shouldIgnoreEvent(event) {
 }
 
 const eventHandler = (callback, options) => (event) => {
-	if (!!options.ctrl !== event.ctrlKey) {
+	if (isMac ? (!!options.ctrl !== event.metaKey) : (!!options.ctrl !== event.ctrlKey)) {
```

Comment:
Having a ternary operator with a condition inside a condition feels hard to read.

```suggestion
	const ctrlKeyPressed = isMac ? event.metaKey : event.ctrlKey
	if (!!options.ctrl !== ctrlKeyPressed) {
```

Reply:
Or we can say that `meta` = `ctrl alias on Mac`, to allow both keys, like search does:
```suggestion
	if (!!options.ctrl !== (event.metaKey || event.ctrlKey)) {
```

Reply:
Maybe the first idea then? on Windows, Win key counts as metaKey, so Win+C would (unexpectedly) work

Reply:
> so Win+C would (unexpectedly) work

I'm shocked